### PR TITLE
Fix camera bugs

### PR DESCRIPTION
**Pull request by ysz0507**
Only fixes:
1. Exit fragment if permissions are denied
2. Start fragment if permissions are granted
3. Make buttons transparent
4. Fix gitignore

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -32,43 +32,43 @@
...
             android:scaleY="1.1"
             android:src="@android:drawable/ic_menu_save"
-            app:tint="#000000"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.504"
-            app:layout_constraintStart_toStartOf="parent" />
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toStartOf="parent"
+            app:tint="#000000" />
```
Comment (QuentinRG) [inline app/src/main/res/layout/activity_main.xml:58]: It seems hard coded, replace with a color.

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -32,43 +32,43 @@
...
+            android:backgroundTint="@color/mainBlue"
             android:scaleType="fitCenter"
             android:src="@android:drawable/ic_dialog_alert"
-            app:tint="#F31919"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toStartOf="@+id/saved_recordings_button"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="@+id/security_camera_button" />
+            app:layout_constraintTop_toTopOf="@+id/security_camera_button"
+            app:tint="#E13232" />
```
Comment (QuentinRG) [inline app/src/main/res/layout/activity_main.xml:42]: It seems hard coded, replace with a color.

**Review by QuentinRG — CHANGES_REQUESTED** [review]
Great work, but some hard coded things

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (ysz0507) [reply app/src/main/res/layout/activity_main.xml:58]: But it's no value to add it to the values. Lets discuss it in person

**Review by QuentinRG — APPROVED** [review]
perfect

**Pull request merged**